sim_pipeline/Pipelines/skypy_pipeline.py

```diff
@@ -1,33 +1,42 @@
 import os
 from skypy.pipeline import Pipeline
 import sim_pipeline
+import tempfile
```

Comment:
'tempfile' is a separate library. can you add it to the requirements.txt file?

Reply:
tempfile is a built-in module, no need to install it via pip I think.